calculator.rb

```diff
@@ -0,0 +1,92 @@
+puts %q{
+"Welcome to the Calculator program! Which operator would you like to use?"
+1. add(+)
+2. subtract(-)
+3. multiply(*)
+4. divide(/)
+5. exponify(**)
+6. modulo(%)
+Please choose one operator(name or symbol):
+}
+operator = gets.chomp
+
+operator_choices = %W[add + subtract - multiply * divide / exponify ** modulo %]
+
+until operator_choices.include?(operator)
+  puts %q{
+  Input error. Please choose from the following:
```

Comment:
The way that you have this formatted means that the puts here is actually indented differently.